doc/tech/usecases.md

```diff
@@ -13,18 +13,32 @@
 ## 5.2 Edge
 Principles:
 
-This section introduces principles that should be followed during the definition and development of the Edge scope for the CNTT Reference Model, Reference Architecture 01 -OpenStack, Reference Architecture 02 - Kubernetes, Reference Implementation and Reference Conformance.
+This section introduces principles that should be followed during the definition and development of the Edge scope for the CNTT Reference Model, Reference Architectures, Reference Implementations and Reference Conformance test suites.
 
-As we are not defining something new, the Edge Principles will follow the [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles), the [Reference Architecture Principles](../ref_arch#principles) and Network Principles. 
+A main principle is that we shall not define a new branch of CNTT and refrain from inventing new things where other organizations already have solutions or ongoing suitable standardization. By that follows that the Edge Principles, as far as possible, shall follow and amend the existing [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles), the [Reference Architecture Principles](../ref_arch#principles) and the Network Principles that are being developed in parallel. 
 
--	Open source preference
--	Open APIs
+There are however a couple of new things that Edge deployments bring into the picture compared to the larger or regional Cloud Infrastructure deployments. These differences will generate a small set of additional principles. For now, there are only a few high level mixed differences and principles listed to generate suitable debate and feedback, but it will be filled out with more clarifications over time and during the development of the prioritized use cases.
+
+-	Distribution into many small sites
+-  Deployment automation
+-  Cloud Infrastructure API accessibility
+-  Automated lifecycle management
+-  Automated scalability
+-  Automated closed loop assurance
+-  On-Site staff trust and competence availability
+-  Security concerns
+-  On-Site access restrictions (distance, accessibility, cost)
+-  Remote analysis, isolation and serviceability
+-  Resource restrictions
+-  Cloud Infrastructure overhead minimization
 -	Separation of concerns
--	Automated lifecycle management
--	Automated scalability
--	Automated closed loop assurance
 -	Cloud nativeness
--	Security compliance:
+-  Geographical presence and data origin
+-  Data locality, protection and regulatory fulfilments
 -	Resilience and Availability
--	Scalability 
+-  WAN connectivity availability, capabilities and quality
+-  Autonomous local infrastructure functionality and operations
+-  Heterogeneous Infrastructure
 -	Workload diversity 
+-  Support of Telco and 3PP applications
```

Comment:
3PP acronym

Reply:
it is 3rd part application, but I tried to edit it but I didn't found it

Reply:
3PP , 3rd party aplication